src/hypersync/DeclarativeProofProvider.ts

```diff
@@ -237,45 +238,49 @@ export class DeclarativeProofProvider extends ProofProviderBase {
     // Proof is always stored as an array.
     const proof = Array.isArray(data) ? data : [data];
 
-    return [
-      {
-        filename: settings.name,
-        contents: {
-          type: process.env.integration_type!,
-          title: resolveTokens(proofSpec.title, tokenContext),
-          subtitle: resolveTokens(proofSpec.subtitle, tokenContext),
-          source: apiUrl,
-          webPageUrl: resolveTokens(proofSpec.webPageUrl, tokenContext),
-          orientation: proofSpec.orientation,
-          userTimeZone: hyperproofUser.timeZone,
-          criteria,
-          proofFormat: settings.proofFormat,
-          template: HypersyncTemplate.UNIVERSAL,
-          layout: {
-            format: proofSpec.format,
-            noResultsMessage:
-              proof.length > 0 || !proofSpec.noResultsMessage
-                ? ''
-                : resolveTokens(proofSpec.noResultsMessage, tokenContext),
-            fields: proofSpec.fields.map(f => ({
-              property: f.property,
-              label: resolveTokens(f.label, tokenContext),
-              width: f.width,
-              type: f.type === HypersyncFieldType.TEXT ? undefined : f.type
-            }))
-          },
-          proof,
-          authorizedUser,
-          collector: this.connectorName,
-          collectedOn: dateToLocalizedString(
-            syncStartDate,
-            hyperproofUser.timeZone,
-            hyperproofUser.language,
-            hyperproofUser.locale
-          )!
+    return {
+      nextPage: response.nextPage,
```

Comment:
Nit: I would like to see these two props after data since a) they aren't used all the time and b) the data is what is most important IMO.

Also, can you remind me how this paging works?  

I think the story is: When paging is used, the data comes back in pages along with a bunch of data that repeats (e.g. the layout info, collectedOn info, etc.).  The job engine takes care of collecting all those chunks.  Then it aggregates all the `proof` elements and extracts the repeating data from one of the chunks (the first one?).  These elements are then combined to form the full proof.

Is that right?  And if so, this is specific to `combine: true`, right?  Do you see a path forward to `combine: false` with the declarative model?

Reply:
That's exactly right. If combine is false, then every `ProofFile` in `data` is turned into separate proof files.

I hard-coded `combine: true` because we don't make proof types with multiple files anymore. I don't think we'll use that option much, and we could consider making that the default

Reply:
If we don't need to support `combine: false` anymore I'd be a fan of getting rid of the option entirely.  OK with a new Jira item to remove somewhere down the road.